go/libraries/doltcore/sqle/enginetest/stats_queries.go

```diff
@@ -331,6 +331,23 @@ var DoltStatsIOTests = []queries.ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "nulls panic bug",
```

Comment:
Can you add more context about what is being tested here? It could be a more detailed name, a comment, or a link to an issue.